packages/iframe-coordinator-cli/package.json

```diff
@@ -10,7 +10,7 @@
     "express": "^4.21.1",
     "find-root": "^1.1.0",
     "http-proxy-middleware": "^3.0.3",
-    "iframe-coordinator": "6.0.0",
```

Comment:
Could we remove this version bump from this PR? I think we should only have the commitlint fix in this PR

Reply:
Sure, I can move that to a new branch. It will make the actions unable to `npm ci` and fail earlier. But it isn't like they are working anyway on this PR.